app/models/user.rb

```diff
@@ -11,6 +11,26 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
+  def friends
```

Comment:
- [ ] **[OPTIONAL]** A better and easier way to define friends and inverse_friends methods is by using `has_many through` associations:
```ruby
has_many :friendships
has_many :confirmed_friends, -> { where(friendships: { friendship_status: true }) }, through: :friendships, source: :friend
has_many :inverse_friendships, class_name: 'Friendship', foreign_key: 'friend_id'
has_many :inverse_confirmed_friends, -> { where(friendships: { friendship_status: true }) }, through: :inverse_friendships, source: :user
```
This is better because active record will be faster than using ruby enumerables.